.github/workflows/release.yml

```diff
@@ -30,13 +32,18 @@ jobs:
           git config user.name "Open Terms Archive Release Bot"
           git config user.email "release-bot@opentermsarchive.org"
           git commit --message="Update changelog" CHANGELOG.md package.json package-lock.json
-          git push origin
+          [ ${{ needs.changelog.outputs.release-type }} != 'no-release' ] && git tag v${{ steps.update-changelog.outputs.version }}
 
-      - name: Tag commit
-        if: needs.changelog.outputs.release-type != 'no-release'
-        run: |
-          git tag v${{ steps.update-changelog.outputs.version }}
-          git push origin --tags
+      - name: Run status checks for release commit on temporary branch  # use temporary branch to enable pushing commits to this branch protected by required status checks
+        uses: CasperWA/push-protected@v2
+        with:
+          token: ${{ secrets.RELEASE_BOT_GITHUB_TOKEN }}
+          branch: main
+          acceptable_conclusions: 'success,skipped'  # changelog checks are skipped when not in a PR towards main branch
```

Comment:
This will become the default value in the next release, see https://github.com/CasperWA/push-protected/pull/250.